packages/config/config/devices/0x0178/9125051.json

```
@@ -30,23 +30,31 @@
 		},
 		"2": {
 			"label": "Discovery Mode",
-			"description": "Start, stop, or get the results of discovery mode. Retrieve the configuration parameter to obtain the results: 0=not enabled, 1=enabled, 2=timed out, 3=failure",
+			"description": "Start, stop, or get the results of discovery mode. Retrieve the configuration parameter to obtain the results: 0=disabled, 1=enabled, 2=Timeout, 3=Failure",
```

```suggestion
			"description": "Start, stop, or get the results of discovery mode. Retrieve the configuration parameter to obtain the results.",
```
We have the labels in the options, no need to repeat them.